test/mocker.test.js

```diff
@@ -2,7 +2,7 @@ const { expect } = require('chai');
 const get = require('lodash.get');
 const { unflatten } = require('flat');
 
-const mongoose3 = require('mongoose3');
+// const mongoose3 = require('mongoose3');
```

Comment:
Hey. Are u not getting lint error about the unused variables here?

Reply:
I am! I commented it out though :) No more errors